src/patterns/by-the-numbers.scss

```diff
@@ -1,228 +1,275 @@
+			column-gap: 1rem;
+			display: grid;
+			grid-template-columns: 1fr 1fr;
+			text-align: center;
+		}
 
-			.wp-block-group__inner-container {
-				height: auto !important;
-				position: relative;
-				z-index: 2;
-			}
+		.wp-block-image {
+			margin-bottom: 0;
+		}
 
-			&__number {
-				font-size: 179px;
-				font-weight: 700;
-				line-height: 1;
-				padding-top: 25%;
-				margin-bottom: 0;
+		&__number {
+			font-size: 58px;
+			font-weight: 700;
+			margin-bottom: 0;
+		}
 
-				@media only screen and ( min-width: 782px ) {
-					padding-top: 10rem;
-				}
-			}
+		&__unit {
+			font-size: 28px;
+			font-weight: 700;
+			margin-bottom: 0;
+		}
 
-			&__text {
-				font-size: $font-size-xlarge;
-				margin: 0 auto 0 auto;
-				width: 60%;
-			}
+		&__text {
+			font-size: 1rem;
+			margin-bottom: 0;
+		}
 
-			figure,
-			svg {
-				height: auto !important;
-				left: 0;
-				position: absolute;
-				top: 0;
-				width: 100%;
-				z-index: 1;
-			}
+		img {
+			margin: auto;
 		}
+	}
 
-		.wmf-pattern-most-visited {
-			margin-top: 0;
+	.wmf-pattern-three-columns {
+		margin-bottom: 7rem;
 
-			svg {
-				width: 170% !important;
-			}
+		@media only screen and ( min-width: 782px ) {
+			margin-bottom: 7rem;
+		}
+
+		@media only screen and ( min-width: 782px ) {
+			margin-bottom: 4rem;
 		}
 
-		.wmf-pattern-most-editors {
+		> .wp-block-group__inner-container {
+			column-gap: 1rem;
+			display: grid;
+			grid-template-columns: 1fr 1fr 1fr;
 			text-align: center;
+		}
 
-			&__number {
-				font-size: 88px;
-				font-weight: 700;
-				margin-bottom: 0;
-			}
+		&__number {
+			font-size: 37px;
+			font-weight: 700;
+			margin-bottom: 0;
+			line-height: 1.3;
 
-			&__text {
-				font-size: $font-size-xlarge;
-				margin: 0 auto 0 auto;
-				width: 60%;
+			@media only screen and ( min-width: 782px ) {
+				font-size: 51px;
 			}
 		}
 
-		.wmf-pattern-two-columns {
+		&__unit {
+			font-size: 17px;
+			font-weight: 700;
+			margin-bottom: 0;
 
-			> .wp-block-group__inner-container {
-				column-gap: 1rem;
-				display: grid;
-				grid-template-columns: 1fr 1fr;
-				text-align: center;
+			@media only screen and ( min-width: 782px ) {
+				margin-bottom: 0.5rem;
 			}
+		}
 
-			.wp-block-image {
-				margin-bottom: 0;
-			}
+		&__text {
+			font-size: 14px;
+			margin-bottom: 0;
+		}
+	}
 
-			&__number {
-				font-size: 58px;
-				font-weight: 700;
-				margin-bottom: 0;
-			}
+	.wmf-pattern-piechart {
+		display: flex;
+		flex-direction: column-reverse;
+		text-align: center;
+		max-width: 330px;
+		margin-bottom: 8rem;
 
-			&__unit {
-				font-size: 28px;
-				font-weight: 700;
-				margin-bottom: 0;
-			}
+		@media only screen and ( min-width: 782px ) {
+			margin-bottom: 4rem;
+		}
 
-			&__text {
-				font-size: 1rem;
-				margin-bottom: 0;
-			}
+		svg {
+			margin-bottom: 0;
+			margin-left: auto;
+			margin-right: auto;
+			max-width: 224px;
 		}
 
-		.wmf-pattern-three-columns {
+		&__number {
+			font-size: 88px;
```

Comment:
Not necessary to change but would be nice to have centralized these rules like

```
.WMF-pattern-a,
.wmf-pattern-b {
    &__numbers {
        font-size: ...
```
If we do this in the future it will make it easier to consistently style the different elements in the same way.